docs/articles/documentation/test-api/test-code-structure.md

```diff
@@ -231,6 +240,82 @@ Each test run has its own test context.
 
 > `t.ctx` is initialized with an empty object without a prototype. You can iterate its keys without the `hasOwnProperty` check.
 
+### Fixture Hooks
+
+Fixture hooks are executed before the first test in a fixture is started and after the last test is finished.
+
+To specify fixture hooks, use the `fixture.before` and `fixture.after` methods.
+
+```text
+fixture.before( fn(ctx) )
+```
+
+```text
+fixture.after( fn(ctx) )
+```
+
+Parameter | Type     | Description
+--------- | -------- | ---------------------------------------------------------------------------
+`fn`      | Function | An asynchronous hook function that contains initialization or clean-up code.
+`ctx`     | Object   | A *fixture context* object used to [share variables](#sharing-variables-between-fixture-hooks-and-test-code) between fixture hooks and test code.
+
+**Example**
+
+```js
+fixture `My fixture`
+    .page `http://example.com`
+    .before( async ctx => {
+        /* fixture initialization code */
+    })
+    .after( async ctx => {
+        /* fixture finalization code */
+    });
+```
+
+#### Sharing Variables Between Fixture Hooks and Test Code
+
+Hook functions passed to `fixture.before` and `fixture.after` methods take a `ctx` parameter that contains *fixture context*.
+You can assign to this parameter to share the value or object with test code.
+
+```js
+fixture `Fixture1`
+    .before(async ctx  => {
+        ctx = 123;
```

Comment:
This is exactly what you **can't do**. You can only assign properties to it, otherwise you just override variable in local scope